src/storagemanager/src/main/java/org/vpac/ndg/task/IProgressCallback.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of the Raster Storage Archive (RSA).
+ *
+ * The RSA is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * The RSA is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * the RSA.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2016 VPAC Innovations
+ * http://vpac-innovations.com.au/
+ */
+
+package org.vpac.ndg.task;
+
+/**
+ * defines an interface that allows a child object to communicate progress without
+ * being too aware of parent.
+ */
+public interface IProgressCallback {
```

Comment:
On Thursday, 23 June 2016, Alex Fraser <notifications@github.com> wrote:

> In
> src/storagemanager/src/main/java/org/vpac/ndg/task/IProgressCallback.java
> <https://github.com/vpac-innovations/rsa/pull/49#discussion_r68154752>:
>
> > + * A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
> > + *
> > + * You should have received a copy of the GNU General Public License along with
> > + * the RSA.  If not, see <http://www.gnu.org/licenses/>.
> > + *
> > + * Copyright 2016 VPAC Innovations
> > + * http://vpac-innovations.com.au/
> > + */
> > +
> > +package org.vpac.ndg.task;
> > +
> > +/**
> > + * defines an interface that allows a child object to communicate progress without
> > + * being too aware of parent.
> > + */
> > +public interface IProgressCallback {
>
> The I prefix is a .NET-ism, and its use is discouraged
> <http://stackoverflow.com/a/542122/320036>. It's better to use proper
> names for interfaces and ~Impl or some other name for implementations.
>
We have an 'ITask', once again just going with what we have.

Also, there's a fine mix of spaces and tabs being used everywhere.



> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vpac-innovations/rsa/pull/49/files/551dfe240d26ab414b21bd50d4dcefe4122b597b#r68154752>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AERMiVBBl-_zs3rmEz5Gy46XQCUUNAsuks5qOceEgaJpZM4I7vPc>
> .
>


-- 
#Lachlan Hurst
Software Developer, Mechanical Engineer, Cyclist
twitter.com/lachlanhurst
facebook.com/lachlan.hurst <http://www.facebook.com/lachlan.hurst>
instagram.com/lachlanhurst
github.com/lachlanhurst